index.js

```diff
@@ -9,7 +9,9 @@ exports.outputFormat = 'js'
 /**
  * Babel's available options.
  */
-const availableOptions = Object.keys(babel.options)
+const availableOptions = Object.keys({
```

Comment:
This will need removing, and the code that currently whitelists options should probably be removed. We should just be able to pass things straight through, I'm not sure if we even need to require plugins of if that's handled for us.

Reply:
@ForbesLindesay ok, i will try to do this

Reply:
@ForbesLindesay Hey! Could you clarify by what it means to remove the code that currently whitelists options here? Maybe I can take a stab at it in a separate PR? I'm interested in upgrading jstransformer-babel to babel7. This would be my very first open-source contribution so I am very excited.